packages/common/src/utils/validateFileName.ts

```diff
@@ -29,6 +29,7 @@ export function isValidFileName(fileName: string) {
   return (
     !INVALID_FILENAME_WHITESPACE_REGEX.test(fileName) &&
```

Comment:
we can restrict the fileName length in the regex in itself. somewhat like `{1,64}`

Reply:
Is that necessary, I would not want to bundle the length restriction into the regex, 
We use regex across the codebsse for mainly character search, baking it in for a one specific case doesn't feel right

Reply:
We can actually use length restrictions in regex. We have done in the scene name regex - https://github.com/EtherealEngine/etherealengine/blob/0e1e55d8809128472211a42e5d4efc4a66df7324/packages/common/src/regex/index.ts/#L33

Reply:
Gotcha

Reply:
implemented